services/config/field_maps.py

```diff
@@ -29,56 +43,129 @@ def handle_strip(value):
     "data-tracker": {
         "view_197": [
             # Signal ID
-            {"src": "field_199_raw", "vza": "field_751", "primary_key": True},
+            {
+                "src": "field_199_raw",
+                "smart-mobility": "field_380",
+                "primary_key": True,
+            },
             # Location Name
-            {"src": "field_211_raw", "vza": "field_624", "handler": handle_strip},
-            # Signal Type
-            {"src": "field_201", "vza": "field_753",},
+            {
+                "src": "field_211_raw",
+                "smart-mobility": "field_382",
+                "handler": handle_strip,
+            },
             # Signal Status
-            {"src": "field_491", "vza": "field_759",},
-            # Location Type
-            {"src": None, "vza": "field_750", "default": "Signal Location",},
-            # Type
-            {"src": None, "vza": "field_626", "default": "Signal"},
-            # Modifie Date/time
-            {"src": "field_205", "vza": "field_647",},
+            {
+                "src": "field_491",
+                "smart-mobility": "field_383",
+            },
+            # Modified Date/time
+            {
+                "src": "field_205",
+                "smart-mobility": "field_391",
+            },
+            # Location ID
+            {
+                "src": "field_209",
+                "smart-mobility": "field_381",
+            },
+            # WEB_URL_GENERATOR_PM
+            {
+                "src": "field_3399",
+                "smart-mobility": "field_384",
```

Comment:
like we talked about, this field is causing the script to detect record changes between the src and dest on every run—so every record is reprocessed every time. it would be best if the knack builder just configured the URL field on their own by mirroring what we do in data tracker (it's a concatenation of an AGOL map and the signal ID).

Reply:
Removing this field map from the ETL and @KaroEngstrom will take it from here!

Reply:
Followed the Data Tracker solution for Artbox: retired the old URL field that had been coming in from Data Tracker; replaced it with text formula and link fields to the ArcGIS map. Tested ✅.